subsys/partition_manager/Kconfig

```diff
@@ -113,6 +113,18 @@ config PM_PGPS_ALIGN
 
 endif
 
+if DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION
+partition=FMFU_STORAGE
+partition-size=DFU_TARGET_FULL_MODEM_EXT_FLASH_SIZE
+rsource "Kconfig.template.partition_config"
+rsource "Kconfig.template.partition_region"
+
+config PM_FMFU_STORAGE_ALIGN
+	hex
+	default $(dt_node_int_prop_hex,$(DT_CHOSEN_ZEPHYR_FLASH),erase-block-size)
```

Comment:
Could you expand a bit on why the erase-block-size of the internal flash is the storage aligner for the external flash?

Reply:
it should not 😅